app/pods/components/my-courses/component.js

```diff
@@ -1,27 +1,72 @@
 import Component from '@ember/component';
 import { inject as service } from '@ember-decorators/service';
 import { restartableTask } from 'ember-concurrency-decorators';
-import { alias } from '@ember-decorators/object/computed';
-import { readOnly } from '@ember-decorators/object/computed';
-
+import { action, computed } from '@ember-decorators/object'
 
 export default class MyCoursesTaskComponent extends Component {
   @service store;
-  @service api;
-  @readOnly("fetchMyCoursesTask.lastSuccessful.value") runs
+	@service api;
+	  
+	limit = 8
+	offset = 8
+	runs = []
+	count = 0
 
-  didReceiveAttrs () {
-    this._super(...arguments)
-    this.get('fetchMyCoursesTask').perform()
-}
+  	constructor () {
+		super(...arguments)
+		this.infiniteScroll = () => {
+			if (document.getElementById('load-more') != null) {
```

Comment:
Let's not use `Load More` button. Instead, check for the condition here.
If there are more runs to load, and user has scrolled to the bottom (use a placeholder element for this), we fetchMoreCourses